webapp-django/crashstats/settings/base.py

```diff
@@ -346,6 +347,12 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 # Rate limit for when using the Web API for anonymous hits
 API_RATE_LIMIT = '10/m'
 
+# Rate limit when using the supersearch web interface
+RATELIMIT_SUPERSEARCH = '5/m'
```

Comment:
Why this super low number? I could see myself making more than 5 searches in a minute, if I'm having to refine a search with facets or columns for example. I think 10 would be a safer default here.